spatialdata/_core/_spatial_query.py

```diff
@@ -85,3 +89,129 @@ def _bounding_box_query_points_dict(
             requested_points[points_name] = points
 
     return requested_points
+
+
+def _bounding_box_query_image(
+    image: Union[MultiscaleSpatialImage, SpatialImage], request: BoundingBoxRequest
+) -> Union[MultiscaleSpatialImage, SpatialImage]:
+    """Perform a spatial bounding box query on an Image or Labels element.
+
+    Parameters
+    ----------
+    image : Union[MultiscaleSpatialImage, SpatialImage]
+        The image element to perform the query on.
+    request : BoundingBoxRequest
+        The request for the query.
+
+    Returns
+    -------
+    query_result : Union[MultiscaleSpatialImage, SpatialImage]
```

Comment:
I don't think we use arguments in `Returns`

Reply:
Should it be like this then?

```python
Returns
-------
The image contained within the specified bounding box.
```

Reply:
Yes I think this is what we use